.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -60,7 +60,7 @@ in the `Generated ApiView` comment added to this PR. You can use ApiView to show
 ## Suppressing failures
 
 If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
-[Swagger-Suppression-Process](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process) 
+[Swagger-Suppression-Process](https://aka.ms/pr-suppressions) 
```

Comment:
Can you please group any and all aka links we create - either under /azsd or /azapi e.g., /azsdk/pr-suppressions? This helps enumerate them for those of us who (co)own hundreds or even hundreds of thousands of aka links.

Reply:
@heaths can you comment on this [here](https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1693256274174?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1693256274174&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1693256274174) ?